src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version.ToString());
```

Comment:
Here there is no call to OS apis, and hence didnt cache, as this operation is relatively simple. I will cache the whole thing as well, although for future coding practices, how do you decide when to cache and when not to? the lifetime of the string is going to be more instead of when it was a local. So the tradeoff will be between execution time and memory. But here both are insignificant.